test/integration/targets/exos_guest_virtual_machines/tests/httpapi/merged.yaml

```diff
@@ -0,0 +1,97 @@
+---
+- debug:
+    msg: "Start exos_guest_virtual_machines merged integration tests on ansible_connection={{ ansible_connection }}"
+
+- include_tasks: _reset_config.yaml
+
+- include_tasks: _populate_config.yaml
+
+- block:
+  - name: Merge the provided configuration with the existing configuration
+    exos_guest_virtual_machines: &merged
+      config:
+        - name: vm_4
+          image: /usr/local/vm/packages/TPVM_Ubuntu16.04_25Feb2019.qcow2
+          memory_size: 2048
+          num_cores: 2
+          vnc:
+            enabled: True
+            port: 5904
+          virtual_ports:
+            - name: VM41-58
+              port: 58
+              type: sriov
+              vlan_id: 40
```

Comment:
Wouldn't we need VLAN 40 to exist for this to work? I don't see it being created in populate_config